.storybook/preview-head.html

```diff
@@ -0,0 +1,8 @@
+<script type="text/javascript">
+  document.addEventListener("DOMContentLoaded", function(event) {
```

Comment:
Wouldn't it be better to mark this script with `defer` instead?

Reply:
I tested it with both defer and async `script.defer = true; ` and `script.async = false;`
![image](https://github.com/qld-gov-au/qg-web-template/assets/126438691/1693e7b1-dcf7-4c80-81c7-a0b8c09c0532)
But it is not functional. I saw DOMContentLoaded being used in a couple of storybook examples on the web.